src/main/java/org/spburegistry/backend/utils/ConvertToTO.java

```diff
@@ -0,0 +1,131 @@
+package org.spburegistry.backend.utils;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.spburegistry.backend.dto.ClientTO;
+import org.spburegistry.backend.dto.ClinicTO;
+import org.spburegistry.backend.dto.CommitTO;
+import org.spburegistry.backend.dto.EducationalProgramTO;
+import org.spburegistry.backend.dto.FacultyTO;
+import org.spburegistry.backend.dto.ProjectTO;
+import org.spburegistry.backend.dto.StudentTO;
+import org.spburegistry.backend.dto.UserTO;
+import org.spburegistry.backend.entity.Client;
+import org.spburegistry.backend.entity.Clinic;
+import org.spburegistry.backend.entity.Commit;
+import org.spburegistry.backend.entity.Project;
+import org.spburegistry.backend.entity.Student;
+import org.spburegistry.backend.entity.User;
+
+public class ConvertToTO {
```

Comment:
Please create private constructor for this class

Reply:
Done